src/Angor/Client/Pages/Wallet.razor

```diff
@@ -235,18 +237,14 @@ else
 
 
                 <button class="btn btn-primary ml-2" @onclick="RefreshBalance">
-                    @if (balanceSpinner)
-                    {
-                        <!-- If balanceSpinner is true, show spinner icon -->
-                        <span class="spinner-border spinner-border-sm opacity-10 btn-content" role="status" aria-hidden="true"></span>
-                    }
-                    else
-                    {
-                        <!-- If balanceSpinner is false, show 'history' icon -->
-                        <span class="material-icons opacity-10 btn-content text-white" data-cy="history-refresh" aria-hidden="true">
-                            history
+
+                    <span data-cy="history-refresh" class="@(balanceSpinner ? "rotate-icon" : "")" aria-hidden="true">
+                            <svg xmlns="http://www.w3.org/2000/svg" width="32px" height="32px" viewBox="0 0 24 24" fill="none">
+                                <path d="M3.67981 11.3333H2.92981H3.67981ZM3.67981 13L3.15157 13.5324C3.44398 13.8225 3.91565 13.8225 4.20805 13.5324L3.67981 13ZM5.88787 11.8657C6.18191 11.574 6.18377 11.0991 5.89203 10.8051C5.60029 10.511 5.12542 10.5092 4.83138 10.8009L5.88787 11.8657ZM2.52824 10.8009C2.2342 10.5092 1.75933 10.511 1.46759 10.8051C1.17585 11.0991 1.17772 11.574 1.47176 11.8657L2.52824 10.8009ZM18.6156 7.39279C18.8325 7.74565 19.2944 7.85585 19.6473 7.63892C20.0001 7.42199 20.1103 6.96007 19.8934 6.60721L18.6156 7.39279ZM12.0789 2.25C7.03155 2.25 2.92981 6.3112 2.92981 11.3333H4.42981C4.42981 7.15072 7.84884 3.75 12.0789 3.75V2.25ZM2.92981 11.3333L2.92981 13H4.42981L4.42981 11.3333H2.92981ZM4.20805 13.5324L5.88787 11.8657L4.83138 10.8009L3.15157 12.4676L4.20805 13.5324ZM4.20805 12.4676L2.52824 10.8009L1.47176 11.8657L3.15157 13.5324L4.20805 12.4676ZM19.8934 6.60721C18.287 3.99427 15.3873 2.25 12.0789 2.25V3.75C14.8484 3.75 17.2727 5.20845 18.6156 7.39279L19.8934 6.60721Z" fill="var(--bs-primary-btn-icon)" />
```

Comment:
Cant this code be cashed in some statis class? other wise the wallet html is bloated